README.md

```diff
@@ -54,10 +54,11 @@ To **find the next item in reading order**, given a reading order item _current_
 1. Let _reading order items_ be the list of reading order items owned by _scope_, sorted in **reading order**.
 2. If _reading order items_ is empty, return null.
 3. If _direction_ is “forward”, then:
-   1. Let _previous_ be the reading order item that comes before _current_, in DOM tree order.
-   2. If _previous_ is null, return the first item in _reading order items_.
-   3. Otherwise, if _previous_ is the last item in _readinging order items_, return null.
-   4. Otherwise, return the item that comes after _previous_ in \_reading order items.
+   1. If _current_ is null, let _previous_ be the last reading order item, in DOM tree order.
```

Comment:
`reading order item from _reading order items_ that comes last in DOM tree order`